frontend/ui/widget/container/framecontainer.lua

```diff
@@ -130,10 +135,11 @@ function FrameContainer:paintTo(bb, x, y)
             self.inner_bordersize, self.color, self.radius)
     end
     if self.bordersize > 0 then
+        local anti_alias = G_reader_settings:nilOrTrue("anti_alias_ui")
         bb:paintBorder(x + self.margin, y + self.margin,
             container_width - self.margin * 2,
             container_height - self.margin * 2,
-            self.bordersize, self.color, self.radius)
+            self.bordersize, self.color, self.radius, anti_alias)
```

Comment:
Is it ok to pass a bool to paintBorder, where it will be passed to paintRoundedCorner() as `anti_alias or 0`, so `true` or `0` - and then to `cblitbuffer.BB_paint_rounded_corner(..., int anti_alias)` as `anti_alias or 0`.
It may work alright, even if a bit non-obvious to grasp, and no need to rebump base - just want somebody else to check all is fine whether we pass initially `anti_alias` as `true` of `false`.

Reply:
It *should* be safe, yeah (c.f., Lua -> C for booleans in http://luajit.org/ext_ffi_semantics.html)

Reply:
Thanks for your double checkings :)